package.json

```diff
@@ -64,11 +64,19 @@
     "utility-types": "^3.4.1"
   },
   "devDependencies": {
+    "@types/enzyme": "^3.9.1",
+    "@types/enzyme-adapter-react-16": "^1.0.5",
+    "enzyme": "^3.9.0",
```

Comment:
Why do we need enzyme? Doesn't react-testing-library cover our use case?

Reply:
I would also have preferred to use react-testing-library. But I had trouble getting it to work with typescript. Its documentation and examples are not as good or mature as enzyme.

Reply:
I think jest along with react-testing-library serves the purpose

Reply:
I like the approach that react-testing library takes to testing. But like I said, it is not as well documented or mature as enzyme is